posthog/settings.py

```diff
@@ -71,11 +71,6 @@ def print_warning(warning_lines: Sequence[str]):
 else:
     JS_URL = os.getenv("JS_URL", "")
 
-PLUGINS_INSTALL_VIA_API = get_from_env("PLUGINS_INSTALL_VIA_API", True, type_cast=strtobool)
-PLUGINS_CONFIGURE_VIA_API = PLUGINS_INSTALL_VIA_API or get_from_env(
```

Comment:
Q: Context on removing these?

Reply:
We've never actually used these. And now we are moving to an approach based on `Organization.plugins_access_level`, which will actually be used.